nixos/modules/config/swap.nix

```diff
@@ -252,6 +252,8 @@ in
           let realDevice' = escapeSystemdPath sw.realDevice;
           in nameValuePair "mkswap-${sw.deviceName}"
           { description = "Initialisation of swap device ${sw.device}";
+            requires = [ "systemd-modules-load.service" ];
```

Comment:
```suggestion
```

systemd-modules-load.service is already required by sysinit.target, so the Requires shouldn't be needed, only the After. (Nothing else in the tree sets Requires=systemd-modules-load.service.)

Reply:
Thanks! I changed it as suggested.